### ISSUE TITLE
A broken `main` ?

### ISSUE THREAD

**Issue by icylace** [jorgebucaran/hyperapp#993]
Shouldn't the `main` prop in package.json be `index.js` instead of `hyperapp.js` ?

https://github.com/jorgebucaran/hyperapp/blob/8e941b1cf37cb5af9032e1aba635229d214b5bb8/package.json#L6

**Comment by jorgebucaran** [comment]
@icylace Maybe. I'm not really sure right now. [`hyperapp.js`](https://unpkg.com/hyperapp@2.0.8/hyperapp.js) is generated in build time and contains Hyperapp as an IIFE. But, why would that be the main entry to the module?

**Comment by icylace** [comment]
What I can say is that as I continue to work with my Hyperapp fork in my own projects as a way of stress-testing my Hyperapp type definitions, I've had to use `index.js` as the `main` so that [Snowpack](https://www.snowpack.dev/) can use my fork properly.

I'm using a direct Github reference to my Hyperapp fork in package.json, so maybe that's significant ?

**Comment by jorgebucaran** [comment]
@icylace `main: index.js` makes sense, `main: hyperapp.js`, doesn't. Let's change it.

**Comment by jorgebucaran** [comment]
Revisiting this now. I can't figure out what's wrong with just using `hyperapp.js`. Either one _should_ work as both are the same code, `hyperapp.js` being the minified version of `index.js`.

**Comment by jorgebucaran** [comment]
Revisiting this just now. I can't figure out what's wrong with using `hyperapp.js`. Either one _should_ work as both contain the same code. `hyperapp.js` is just the minified version of `index.js`.

**Comment by jorgebucaran** [comment]
After fiddling a bit more with this I ended up removing the build and minify scripts (82a0170). Since Hyperapp is provided as an ES module and `pkg.main` points to `index.js` now (20b9a45), minifying Hyperapp is currently doing absolutely nothing.

There isn't much point to minify even for the benefit of CDN usage (which was the one intention behind minifying since we went head-first with ES modules) given that all of the source is already in a single file, written in JavaScript, and very small.
